hw/syn/tools/dc/parse-syn-report.py

```diff
@@ -17,20 +17,20 @@
 CROSSCHECK_REL_TOL = 0.001
 
 
-def _match_strings(full_file, master_key, patterns, results):
+def _match_strings(full_file, primary_key, patterns, results):
```

Comment:
I'm not keen on this name change, because "primary key" makes me think of databases. I think there's a simple refactoring in the Python code that gets rid of this argument completely: I'll make a PR in a minute with that - maybe it would be a better approach?

Probably leave these changes in for now and our PRs can "race" - the slower PR gets to rebase :-)

Reply:
Proposed refactoring in #2756.